mypy.ini

```diff
@@ -1,5 +1,44 @@
 [mypy]
-ignore_missing_imports = True
 
 [mypy-API.migrations.*]
+# Auto-generated code
```

Comment:
This comment confuses me, because I usually write a comment like `# AUTO-GENERATED` in code that my scripts generate. I acknowledge that this is mostly a me problem! :joy:

How about...

```suggestion
# Do not type-check Django migrations (automatically created by Django)
```